en/docs/manage/api-security.md

```diff
@@ -1,4 +1,4 @@
 # API Security
 
 !!! note
-    This section will  discuss about API security in terms of Authentication, Authorization,and  access control. 
+    This section will describe API security in terms of Authentication, Authorization, and Access Control. 
```

Comment:
Why are we capitalising the first letter for Authentication, Authorisation, and Access Control?

Reply:
In the original text, "Authentication" and "Authorization" was capitalized, but not "Acccess Control". Capitalized "Access Control" to make it consistent. Shall we either capitalize all or have all in simple case?